soql-server-pg/src/main/scala/com/socrata/pg/server/analyzer2/ProcessQuery.scala

```diff
@@ -164,6 +171,7 @@ object ProcessQuery {
     // Our etag will be the hash of the inputs that affect the result of the query
     val etag = ETagify(
       analysis.statement.schema.valuesIterator.map(_.name).toSeq,
+      hintsByName,
```

Comment:
Am surprised you're making hints affect the ETag. I don't expect they'll matter often, but I had assumed you weren't going to let them change the results.

Reply:
The etag is basically a hash of the result (implemented here as a hash of all the things that can affect the result) - since the hints affect the result, they go in the big etag mixer.